### PR TITLE
857201: Wirte UG documentation for the paging support.

### PR DESCRIPTION
**Pull request by RasikaPalanisamy**
Created the UG for MAUI SfDataPager.

### CODE REVIEW

File: MAUI/DataGrid/paging.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+The following picture shows the customize styles of data pager:
+
+![DataPager style .NET MAUI DataGrid](Images\paging\maui-datagrid-pager-style.png)
+
+### Limitations
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:339]: Need to add topics that cover all the design spec properties in the paging UG.

- Events
- Display mode
- Ellipsis mode
- Move to pages programmatically
- Display template to the navigation buttons
- Orientation

File: MAUI/DataGrid/paging.md [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    <local:DataPagerViewModel x:Name="viewModel" />
+</ContentPage.BindingContext>
+  
+<ContentPage.Content>
+    <Grid> 
+      <Grid.RowDefinitions>
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+      </Grid.RowDefinitions>
+      <pager:SfDataPager x:Name ="dataPager"
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:56]: We bind the DataGrid ItemsSource from DataPager PagedSource property. So, we need to initialize the SfDataPager before the SfDataGrid.

File: MAUI/DataGrid/paging.md [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+<ContentPage.BindingContext>
+    <local:DataPagerViewModel x:Name="viewModel" />
+</ContentPage.BindingContext>
+  
+<ContentPage.Content>
+    <Grid> 
+      <Grid.RowDefinitions>
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+      </Grid.RowDefinitions>
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:142]: We bind the DataGrid ItemsSource from DataPager PagedSource property. So, we need to initialize the SfDataPager before the SfDataGrid.

File: MAUI/DataGrid/paging.md [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The following screenshot shows the outcome upon execution of the above code:
+
+![Normal paging .NET MAUI DataGrid](Images\paging\maui-datagrid-normal-paging.png)
+
+N> The `SfDataPager` provides scrolling animation while tapping the `FirstPageButton` or `LastPageButton`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+
+## OnDemandPaging	
+
+In normal Paging, data collection is entirely loaded initially into the `SfDataPager`. However, the control also allows for dynamically loading the data for the current page by setting `SfDataPager.UseOnDemandPaging` to `true`.
+
+To load the current page item dynamically, hook into the `OnDemandLoading` event. In the `OnDemandLoading` event, use the `LoadDynamicItems` method to load data for the corresponding page in the `SfDataPager`.
+
+The `OnDemandLoading` event is triggered when the pager moves to the corresponding page. It contains the following event arguments:
+
+ * `StartIndex`: Displays the corresponding page start index.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+		grid.Children.Add(dataGrid);
+		grid.Children.Add(pager);
+		grid.SetRow(dataGrid, 0);
+		grid.SetRow(pager, 1);
+		this.Content = grid;
+	}
+
+  private void dataPager_OnDemandLoading(object sender, OnDemandLoadingEventArgs e)
+  {
+     pager.LoadDynamicItems(e.StartIndex, viewModel.OrdersInfo.Skip(e.StartIndex).Take(e.PageSize));
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:194]: Checked the code and its working fine.

File: MAUI/DataGrid/paging.md [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Button size and Font size of .NET MAUI DataGrid](Images\paging\maui-datagrid-buttonsize-and-fontsize.png)
+
+## DataPagerStyle
+
+The data grid allows you to change its appearance by creating a style class that overrides the `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 1 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:577]: Added the mentioned topics.

File: MAUI/DataGrid/paging.md [hunk 8]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+        dataPager.AutoEllipsisMode = DataPagerEllipsisMode.After;
+        dataPager.AutoEllipsisText = "***";
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Move to pages programmatically
+
+### MoveToFirstPage
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:376]: Move to the first page

File: MAUI/DataGrid/paging.md [hunk 9]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the first page using the `MoveToFirstPage()` method.
+
+### MoveToLastPage
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:380]: Move to the last page

File: MAUI/DataGrid/paging.md [hunk 10]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the last page using the `MoveToLastPage()` method.
+
+### MoveToNextPage
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:384]: Move to to the next page

File: MAUI/DataGrid/paging.md [hunk 11]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the next page using the `MoveToNextPage()` method.
+
+### MoveToPreviousPage
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:388]: Move to to the previous page

File: MAUI/DataGrid/paging.md [hunk 12]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the previous page using the `MoveToPreviousPage()` method.
+
+### MoveToPage(Int32)
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:392]: Move to page

File: MAUI/DataGrid/paging.md [hunk 13]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the desired page using the `MoveToPage(Int32)` method.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:394]: The data pager allows the users to programmatically navigate to the desired page using the `MoveToPage(Int32)` method. Users can also able to navigate to the page with animation using the `MoveToPage(Int32, Int32, Boolean)` method

File: MAUI/DataGrid/paging.md [hunk 14]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+### MoveToPage(Int32, Int32, Boolean)
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:396]: This is not required.

File: MAUI/DataGrid/paging.md [hunk 15]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the users to programmatically navigate to the desired page from the current page using the `MoveToPage(Int32, Int32, Boolean)` method.
+
+## Orientation
+
+By default, `SfDataPager` scrolls horizontally. However, `SfDataPager.Orientation` property allows users to customize the pager to scroll vertically or horizontally according to their preferences.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:402]: By default, `SfDataPager` displays the button in the horizontal direction. However, the `SfDataPager.Orientation` property allows users to customize the data pager to display the buttons vertically or horizontally according to their preferences.

File: MAUI/DataGrid/paging.md [hunk 16]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+        InitializeComponent();
+        dataPager.Orientation = DataPagerScrollOrientation.Vertical;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Events
+
+### PageChanged
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:426]: PageChanging event should be written first.

File: MAUI/DataGrid/paging.md [hunk 17]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+    private void DataPager_PageChanging(object sender, Syncfusion.Maui.DataGrid.DataPager.PageChangingEventArgs e)
+    {
+        // you can get the old page index and new page index here.
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
 ## DataPagerStyle
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:477]: Customize the appearance

File: MAUI/DataGrid/paging.md [hunk 18]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+<td> {{'`NavigationButtonIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation buttons.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color for numeric button.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonSelectionBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets background color of the selection numeric button.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:514]: Gets or sets the background color of the numeric button that is currently selected.

File: MAUI/DataGrid/paging.md [hunk 19]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`NumericButtonSelectionTextColor`'| markdownify }} </td>
+<td> Gets or sets the foreground color of the selection numeric button.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:518]: Gets or sets the text color of the numeric button that is currently selected.

File: MAUI/DataGrid/paging.md [hunk 20]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`NumericButtonTextColor`'| markdownify }} </td>
+<td> Gets or sets the foreground color of the numeric button.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:522]: Gets or sets the text color of the numeric buttons.

File: MAUI/DataGrid/paging.md [hunk 21]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`FirstPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the first page button template.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:526]: Gets or sets the template for the first page navigation button.

File: MAUI/DataGrid/paging.md [hunk 22]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`LastPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the last page button template.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:530]: Gets or sets the template for the last page navigation button.

File: MAUI/DataGrid/paging.md [hunk 23]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`NextPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the next page button template.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:534]: Gets or sets the template for the next page navigation button.

File: MAUI/DataGrid/paging.md [hunk 24]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+</tr>
+<tr>
+<td> {{'`PreviousPageButtonTemplate`'| markdownify }} </td>
+<td> Gets or sets the template of the previous page button template.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:538]: Gets or sets the template for the previous page navigation button.

File: MAUI/DataGrid/paging.md [hunk 25]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+<td> {{'`NavigationButtonDisableIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation button when it is in a disabled state.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation buttons.</td>
+</tr>
+<tr>
+<td> {{'`NumericButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color for numeric button.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:510]: Gets or sets the background color for the numeric buttons.

File: MAUI/DataGrid/paging.md [hunk 26]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+<td> {{'`NavigationButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonDisableBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button when it is in a disabled state.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonDisableIconColor`'| markdownify }} </td>
+<td> Gets or sets the icon color of the navigation button when it is in a disabled state.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:502]: Gets or sets the icon color of the navigation buttons when it is disabled.

File: MAUI/DataGrid/paging.md [hunk 27]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+<td> {{'`DataPagerBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the SfDataPager.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonDisableBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button when it is in a disabled state.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:498]: Gets or sets the background color of the navigation buttons when it is disabled.

File: MAUI/DataGrid/paging.md [hunk 28]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> {{'`DataPagerBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the SfDataPager.</td>
+</tr>
+<tr>
+<td> {{'`NavigationButtonBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the navigation button.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:494]: Gets or sets the background color of the navigation buttons.

File: MAUI/DataGrid/paging.md [hunk 29]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+The `SfDataPager` allows customizing the appearance using the following properties.
+
+<table>
+<tr>
+<th> Property </th>
+<th> Description </th>
+</tr>
+<tr>
+<td> {{'`DataPagerBackgroundColor`'| markdownify }} </td>
+<td> Gets or sets the background color of the SfDataPager.</td>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:490]: Gets or sets the background color of the `SfDataPager`.

File: MAUI/DataGrid/paging.md [hunk 30]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+}
+{% endhighlight %}
+{% endtabs %}
+
 ## DataPagerStyle
 
-The data grid allows you to change its appearance by creating a style class that overrides the `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
+The data pager allows you to change its appearance by changing the properties of `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.
+
+The `SfDataPager` allows customizing the appearance using the following properties.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/paging.md:481]: The data pager allows you to change its appearance by modifying the properties of `DataPagerStyle` and then assigning it to the `SfDataPager.DefaultStyle` property.

The `SfDataPager` enables customization of its appearance using the following properties:

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

File: MAUI/DataGrid/paging.md [hunk 31]
```diff
@@ -106,8 +106,6 @@ The following screenshot shows the outcome upon execution of the above code:
 
 ![Normal paging .NET MAUI DataGrid](Images\paging\maui-datagrid-normal-paging.png)
 
-N> The `SfDataPager` provides scrolling animation while tapping the `FirstPageButton` or `LastPageButton`.
-
 ## OnDemandPaging
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 32]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+    public MainPage()
+    {
+        InitializeComponent();
+        dataPager.DisplayMode = DataPagerDisplayMode.FirstLastNumeric;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## AutoElipsisMode
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 33]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
 
 ![Button size and Font size of .NET MAUI DataGrid](Images\paging\maui-datagrid-buttonsize-and-fontsize.png)
 
+## DisplayMode
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 34]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
 
 ![Button size and Font size of .NET MAUI DataGrid](Images\paging\maui-datagrid-buttonsize-and-fontsize.png)
 
+## DisplayMode
+
+The `SfDataPager.DisplayMode` property enables the user to personalize the buttons displayed on the datapager. By default, all buttons (including first, last, next, previous, and numeric buttons) are visible.
+
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:356]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 35]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
 
 ![Button size and Font size of .NET MAUI DataGrid](Images\paging\maui-datagrid-buttonsize-and-fontsize.png)
 
+## DisplayMode
+
+The `SfDataPager.DisplayMode` property enables the user to personalize the buttons displayed on the datapager. By default, all buttons (including first, last, next, previous, and numeric buttons) are visible.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 27 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 26 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 36]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+        InitializeComponent();
+        dataPager.DisplayMode = DataPagerDisplayMode.FirstLastNumeric;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## AutoElipsisMode
+
+The `AutoEllipsisMode` is a feature that determines the ellipsis mode for the numeric buttons within the scrollable panel of the SfDataPager. By default, all numeric buttons on the page are visible. However, you have the option to adjust the auto ellipsis mode using the `SfDataPager.AutoEllipsisMode` property.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 37]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+        InitializeComponent();
+        dataPager.AutoEllipsisMode = DataPagerEllipsisMode.After;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![AutoEllipsisMode .NET MAUI DataGrid](Images\paging\maui-datagrid-autoelipsismode.png)
+
+## AutoEllipsisText
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 38]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+
+The data pager allows the user to customize the text to be displayed in the auto ellipsis mode by using `SfDataPager.AutoEllipsisText` property. The default value of `AutoEllipsisText` is set to `…`.
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md [hunk 39]
```diff
@@ -297,9 +295,249 @@ public partial class MainPage : ContentPage
...
+    {
+        InitializeComponent();
+        dataPager.AutoEllipsisMode = DataPagerEllipsisMode.After;
+        dataPager.AutoEllipsisText = "***";
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Move to pages programmatically
```
[the remark this answers is not in the document]
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 8 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 9 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 10 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 11 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 12 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 13 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 14 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 16 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 17 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 18 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 19 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 20 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 21 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 22 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 23 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 24 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 25 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 28 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 29 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md:541]: Addressed the concerns provided.

File: MAUI/DataGrid/paging.md (hunk 30 above)
Comment (RasikaPalanisamy) [reply MAUI/DataGrid/paging.md]: Addressed the concerns provided.

**Pull request merged**